docs/data_repository_template.md

```diff
@@ -0,0 +1,81 @@
+.. _data-repo-template:
+
+# Scivision data repository template
+
+In order for `scivision` to be able to load a dataset and run a computer vision (CV) model on it, a GitHub repository containing configuration for one or more datasets must first be created.
+
+This guide explains how to set up a GitHub repository for your datasets compatible with scivision.
+
+This is also a pre-requisite for adding the dataset to the scivision "catalog", enabling other users of scivision to use it. To learn how to do this, consult the [contributor page](../contributing.md#gift-extending-the-scivision-catalog) after setting up your data repo as per this guide.
+
+## 📚 Contents
+
+- :ref:`data-repo-structure`
+- :ref:`data-config-file`
+- :ref:`documentation`
+- :ref:`data-license`
+- :ref:`example-data-repos`
+
+.. _data-repo-structure:
+
+## 🧱 Data repo structure
+
+We recommend to set up a data repo for scivision with the following structure, with the default name for the config file that the scivision API will recognise being `.scivision/data.yml`:
+
+```
+exampleuser/data_repo
+│   README           
+│   LICENSE          
+│   
+└───.scivision
```

Comment:
Is `.scivision` handled specially by anything at the moment? (Question just for my own clarification - I think fine even if we still need to give the full path for now)

Reply:
yes, the default configs are called `.scivision/data.yml` or `.scivision/model.yml` meaning that if the path to the data/model repo is provided to load_dataset/load_pretrained_model but not the full path to config, it assumes they are located/named like that